target/linux/ipq40xx/files-5.4/drivers/net/phy/qca807x.c

```diff
@@ -314,7 +322,7 @@ static int qca807x_gpio_get_direction(struct gpio_chip *gc, unsigned int offset)
 
 static int qca807x_gpio_get_reg(unsigned int offset)
 {
-	return QCA807X_MMD7_LED_100N + offset % 4;
+	return QCA807X_MMD7_LED_100N_1 + offset % 4;
```

Comment:
Only `QCA807X_MMD7_LED_100N_2` and `QCA807X_MMD7_LED_1000N_2` are used to force LED output, so it should be like
```c
return QCA807X_MMD7_LED_100N_2 + (offset % 2) * 2;
```

Reply:
Will correct those.